### DISCUSSION TITLE
Should we start a place to collect flix packages?

### DISCUSSION THREAD

**Discussion by JonathanStarup** [flix/flix#5130] [Ideas]
They're hard to find otherwise and if we crate it we can have some control over it

**Comment by magnus-madsen** [comment]
It would be nice to just start with a ticket that lists all of our packages.

**Comment by jaschdoc** [comment]
Maybe we could just support git repositories as the default way of publishing? I think Go does that

**Reply by magnus-madsen** [reply]
We already support that. https://doc.flix.dev/packages.html
